index.html

```diff
@@ -425,10 +425,16 @@ <h4>The <dfn>ThingFilter</dfn> dictionary</h4>
             Let <var>model</var> be the result of running the <a>TD parsing algorithm</a> with <var>td</var> as argument. If that throws an error, re-throw the error and terminate these steps.
```

Comment:
"model" is later used for either TD string or ThingFragment object.
Thus, I would call this here "fragment", as it is an object. The "TD parsing algorithm" could already take care of the default values (see also below).